apps/api/src/services/enrollment-history.ts

```diff
@@ -159,7 +159,20 @@ export class EnrollmentHistoryService {
         section.statusHistory.push(row.status ?? "");
       }
     }
-    return transformedSectionRows
+    const filteredSections = Array.from(transformedSectionRows.values()).filter((section) => {
```

Comment:
The newest versions of node added some methods from `Array` onto `Iterator`, so we can preform `filter` directly on `transformedSectionRows.values()`

```suggestion
    const filteredSections = transformedSectionRows.values().filter((section) => {
```